core/domain/wipeout_service.py

```diff
@@ -1213,64 +1227,68 @@ def _remove_user_id_from_models_transactional(
                 i:i + feconf.MAX_NUMBER_OF_OPS_IN_TRANSACTION])
 
 
-def _pseudonymize_app_feedback_report_models(
-    pending_deletion_request: wipeout_domain.PendingDeletionRequest
+def _pseudonymize_one_model_class(
+    pending_deletion_request: wipeout_domain.PendingDeletionRequest,
+    model_class: Type[base_models.BaseModel],
+    property_containing_user_id_name: str,
+    module_name: models.Names
 ) -> None:
-    """Pseudonymize the app feedback report models for the user with user_id,
-    if they scrubbed a feedback report. If the user scrubs multiple reports,
-    they will be given the same pseudonym for each model entity.
+    """Pseudonymize one model class for the user with user_id.
```

Comment:
... with the given user_id

But I'm a bit confused here, no user ID is actually given in the args. Could you clarify?

Edit: Ah, I would suggest "... with the user_id associated with the given pending deletion request." (In general, docstrings should make it clear how the function output relates to its inputs, so parts like this need to be clarified.)

Reply:
Done.